draft-meynell-panrg-scion-research-questions.md

```diff
@@ -471,11 +473,18 @@ Is reverse path refresh a relevant problem?
 -->
 
 # Interfaces for Path Awareness
+The questions posed here are:
```

Comment:
If we refer to https://www.rfc-editor.org/rfc/rfc9217.html, I think `Interfaces for Path Awareness`  are something different than what is discussed here. I think they mean interfaces for applications/users to select paths.

However, this chapter discusses more the interfacing of SCION with existing internet protocols/solution.

Maybe rename this section to `Interfacing SCION with existing technologies`?